megamek/src/megamek/common/MechSummaryCache.java

```diff
@@ -245,9 +274,23 @@ public void loadMechData(boolean ignoreUnofficial) {
             bNeedsUpdate |= loadMechsFromDirectory(vMechs, sKnownFiles, lLastCheck, userDataUnits, ignoreUnofficial);
         }
 
+        // save updated cache back to disk
+        if (bNeedsUpdate) {
+            try {
+                saveCache();
+            } catch (Exception e) {
+                loadReport.append("  Unable to save mech cache\n");
```

Comment:
```suggestion
                loadReport.append(" Unable to save mech cache\n");
```

Reply:
Double spacing is an accessibility issue. Further, I just want to check that the initial space is correct, as in general that's an odd way to handle it.